db/bigtable.go

```diff
@@ -701,152 +890,170 @@ func (bigtable *Bigtable) SaveSyncComitteeDuties(blocks map[uint64]map[string]*t
 		logger.Infof("no sync duties to export")
 		return nil
 	}
+
+	muts := make([]*gcp_bigtable.Mutation, 0, utils.Config.Chain.Config.SlotsPerEpoch*utils.Config.Chain.Config.SyncCommitteeSize+1)
+	keys := make([]string, 0, utils.Config.Chain.Config.SlotsPerEpoch*utils.Config.Chain.Config.SyncCommitteeSize+1)
+
 	for slot, validators := range dutiesBySlot {
-		mut := gcp_bigtable.NewMutation()
+		participation := uint64(0)
 		for validator, participated := range validators {
+			mut := gcp_bigtable.NewMutation()
 			if participated {
-				mut.Set(SYNC_COMMITTEES_FAMILY, fmt.Sprintf("%d", validator), gcp_bigtable.Timestamp((max_block_number-slot)*1000), []byte{})
+				mut.Set(SYNC_COMMITTEES_FAMILY, "s", gcp_bigtable.Timestamp((MAX_BLOCK_NUMBER-slot)*1000), []byte{})
```

Comment:
While with current mainnet config of 1 slot every 12 seconds this is save for a couple hundred years, MAX_BLOCK_NUMBER-slot could be potentially negative for a chain with lower slot times (potential L2 maybe).

Reply:
even with chains with a block time of one second this should last around 30 years